projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -39,6 +39,13 @@ public function dequeue_scripts() {
 		wp_dequeue_script( 'a8c_wpcom_masterbar_overrides' ); // Initially loaded in modules/masterbar/masterbar/class-masterbar.php.
 	}
 
+	/**
+	 * Determines whether the current locale is right-to-left (RTL).
```

Comment:
I wonder if it may make sense to expand on that description a bit, to outline the differences between the site's settings and the linked WordPress.com account settings?

Reply:
Yup, makes sense. Added in fb884e3.